Cargo.toml

```diff
@@ -21,22 +21,24 @@ stream = []
 log = "0.4"
 futures-util = { version = "0.3", default-features = false, features = ["async-await", "sink", "std"] }
 pin-project = "1.0"
-tokio = { version = "0.3", default-features = false, features = ["io-util"] }
+tokio = { version = "1.0.0", default-features = false, features = ["io-util"] }
 
 [dependencies.tungstenite]
 version = "0.11.1"
 default-features = false
+git = "https://github.com/nickelc/tungstenite-rs.git"
+branch = "bytes"
```

Comment:
```suggestion
git = "https://github.com/snapview/tungstenite-rs.git"
rev = "208061ba284eaf95a54c1b9b6968f570004c90de"
```

Reply:
Thanks for notifying. I saw that it was merged today morning but thought to wait until the new version is released. Because I thought the snapview team probably won't want to release a new version of `tokio-tungstenite` with a "repo-based" dependency but official versions instead 🤔.

I'll update to the master branch then as you suggested 🙇‍♂️.

Reply:
Yeah, I figure they will want to use an official version too, this will just keep it working in the meantime :)